tox.ini

```diff
@@ -9,17 +9,38 @@ deps =
     flake8    
     isort    
     pytest
+    pytest-cov
 
 commands =    
-    black --check ccn-template
-    isort --check ccn-template
-    flake8 ccn-template --max-complexity 10
-    #pytest tests/
-    #coverage run --source=ccn-template --branch -m pytest tests/
-    #coverage report -m
+    black --check src
+    isort --check src
+    flake8 --config={toxinidir}/tox.ini src
+    pytest
```

Comment:
or are they automatically included in pytest? (they seemed to have run when I called `tox`)

Reply:
I added a comment in the pyproject.toml to clarify the initial configuration of pytest

Reply:
I'm confused -- now we have `--cov=ccn_template` in both pyproject.toml and tox.ini, doesn't it only need to be in one (pyproject), with a comment in tox.ini saying coverage will automatically be run because of pyproject.toml?

Reply:
yes, it should. I'll change that